test/unit/org/apache/cassandra/cql3/ast/Mutation.java

```diff
@@ -161,22 +166,38 @@ public Stream<? extends Element> stream()
         {
             return Stream.of(value);
         }
+
+        public long get()
+        {
+            if (value.value() instanceof Long)
+                return (long) value.value();
+            return LongType.instance.compose(value.valueEncoded());
+        }
     }
 
     public static class Using implements Element
     {
         public final Optional<TTL> ttl;
         public final Optional<Timestamp> timestamp;
 
-        public Using(Optional<TTL> ttl, Optional<Timestamp> timestamp)
+        private Using(Optional<TTL> ttl, Optional<Timestamp> timestamp)
         {
             this.ttl = ttl;
             this.timestamp = timestamp;
+            if (ttl.isEmpty() && timestamp.isEmpty())
```

Comment:
it was possible to have a `Using` with everything empty, which adds extra spacing when you pretty print the CQL... removed this case to keep things simple; this was found in `Batch` and isn't 100% for this patch

Reply:
Does this break existing CQL in a way we care about? Was this already an error somewhere else in C*?

Reply:
this isn't CQL parsing code, its an isolated `ast` that is used to generate CQL.  This can't break production